src/sentry/testutils/fixtures.py

```diff
@@ -651,17 +651,22 @@ def create_uptime_subscription(
         subscription_id: str | None = None,
         status: UptimeSubscription.Status = UptimeSubscription.Status.ACTIVE,
         url="http://sentry.io/",
+        host_provider_id: str | None = None,
         interval_seconds=60,
         timeout_ms=100,
         date_updated: None | datetime = None,
     ) -> UptimeSubscription:
         if date_updated is None:
             date_updated = timezone.now()
+        if host_provider_id is None:
+            host_provider_id = "TEST"
```

Comment:
Just noting that this makes it hard to create one of these with an actual `None` value here